src/features/identityCheck/useIdentityCheckSteps.tsx

```diff
@@ -4,17 +4,24 @@ import React from 'react'
 import { IdentityCheckMethod } from 'api/gen'
 import { useIdentityCheckContext } from 'features/identityCheck/context/IdentityCheckContextProvider'
 import { IdentityCheckStep, StepConfig } from 'features/identityCheck/types'
+import { env } from 'libs/environment'
 import { Confirmation } from 'ui/svg/icons/Confirmation'
 import { IdCard } from 'ui/svg/icons/IdCard'
 import { Profile } from 'ui/svg/icons/Profile'
+import { Smartphone } from 'ui/svg/icons/Smartphone'
 import { IconInterface } from 'ui/svg/icons/types'
 
 // hook as it can be dynamic depending on subscription step
 export const useIdentityCheckSteps = (): StepConfig[] => {
   const { profile, identification } = useIdentityCheckContext()
   const hasSchoolTypes = profile.hasSchoolTypes
 
-  return [
+  /**
+   * TODO: Change to feature flag
```

Comment:
Tu peux mettre un message un peu plus explicite et l'associer au numéro de ticket sur lequel on va résoudre le problème ?

ex : `TODO: (PC-14455) replace this const with result from stepperIncludesPhoneValidation value from useNextSubscriptionStep`